test/dlc_tests/container_tests/bin/testSmdebug

```diff
@@ -74,6 +74,13 @@ elif [[ $FRAMEWORK == "mxnet" ]]; then
         exit $exit_code
     fi
 else
+    if [[ $FRAMEWORK == "tensorflow2" ]]; then
+        framework_version=$(python -c "import tensorflow; print(tensorflow.__version__)")
+        if [[ $framework_version == *"2.0."* ]]; then
+            # Installing tensorflow-datasets==3.2.1 here prevents future "pip install"s from installing higher versions
+            pip install tensorflow_datasets==3.2.1
+        fi
+    fi
     pip install tensorflow-datasets
```

Comment:
I think you just need to do pip install tensorflow-datasets==3.2.1 here, probably and get rid of above code. 
Other ise this line would override.

Reply:
Hey, I checked if that happens. If you run `pip install` without a `--upgrade` or `-U`, then it keeps whatever version has already been installed.

This was intentional. But you're right, it may be confusing when read. I'll update it.